int8[2] point
